locales-src/onboarding.ftl

```diff
@@ -30,6 +30,29 @@ onboarding-join-form-continue = Continue
 
 onboarding-start-browsing-button-label = Start Browsing
 
+## Welcome modal dialog strings for pre-trailhead about welcome.
+
+onboarding-control-welcome-header = Take { -brand-product-name } with You
+onboarding-control-welcome-content = Get your bookmarks, history, passwords and other settings on all your devices.
+onboarding-control-welcome-learn-more-link = Learn more about { -brand-product-name } Accounts
```

Comment:
"Firefox Account" is a brand.. although this one is plural… Then again, perhaps it shouldn't be? @flodolo ?

https://searchfox.org/mozilla-central/source/browser/locales/en-US/browser/branding/sync-brand.ftl#13

Reply:
I guess we could do something like `{ -fxaccount-brand-name(case: plural) }` and update sync-brand ?

Reply:
Touching the branding file is not an option. You would update English, and locales won't even know you did, unless you break half of Firefox.

I believe this shouldn't be plural in the first place.

```
onboarding-control-welcome-learn-more-link = Learn more about { -fxaccount-brand-name }
```

Reply:
Let's just use the original string as is to avoid needing new translations. Seems likely we won't be using this string soon anyway if we end up using the new join modal messaging.